From 90e017fdde30d51f39325a7b2daea9459ba2c6c6 Mon Sep 17 00:00:00 2001
From: Olivier Moysan <olivier.moysan@foss.st.com>
Date: Thu, 22 Feb 2024 10:57:50 +0100
Subject: [PATCH] arm64: defconfig: add stm32 lptimer support in addons
 fragment

Enable LPTimer drivers that can be used on STM32MP25, as module.

Signed-off-by: Fabrice Gasnier <fabrice.gasnier@foss.st.com>
Signed-off-by: Olivier Moysan <olivier.moysan@foss.st.com>
Change-Id: I8baaadab09d8b2438ea1e1a00f1b1e88a326be93
Reviewed-on: https://gerrit.st.com/c/mpu/oe/st/linux-stm32/+/361425
ACI: CITOOLS <MDG-smet-aci-reviews@list.st.com>
---
 arch/arm64/configs/fragment-02-defconfig-addons.config | 5 +++++
 1 file changed, 5 insertions(+)

--- a/arch/arm64/configs/fragment-02-defconfig-addons.config
+++ b/arch/arm64/configs/fragment-02-defconfig-addons.config
@@ -1,14 +1,19 @@
 #  ALPHABETICALLY SORTED: $LC_ALL=C sort fragment-02-defconfig-addons.config -o fragment-02-defconfig-addons.config
+CONFIG_COUNTER=m
 CONFIG_DYNAMIC_DEBUG=y
+CONFIG_IIO_STM32_LPTIMER_TRIGGER=m
 CONFIG_IIO_STM32_TIMER_TRIGGER=m
+CONFIG_MFD_STM32_LPTIMER=m
 CONFIG_MFD_STM32_TIMERS=m
 CONFIG_OF_OVERLAY=y
 CONFIG_PWM_STM32=m
+CONFIG_PWM_STM32_LP=m
 CONFIG_REGULATOR_ARM_SCMI=y
 CONFIG_REMOTEPROC=y
 CONFIG_STM32_ADC=m
 CONFIG_STM32_ADC_CORE=m
 CONFIG_STM32_RPROC=y
+CONFIG_STM32_LPTIMER_CNT=m
 CONFIG_STM32_TIMER_CNT=m
 CONFIG_TEE_REMOTEPROC=y
 CONFIG_USB_HIDDEV=y
